pkg/jobs/utils.go

```diff
@@ -60,7 +107,14 @@ ORDER BY created`
 			return false /* exists */, err
 		}
 
-		if payloadPredicate(payload) {
+		isTyp := false
+		for _, typ := range jobTypes {
+			if payload.Type() == typ {
+				isTyp = true
+			}
+			break
```

Comment:
should this break be inside the if condition? I assume we want to check the payload type against all `jobTypes` before bailing.

Reply:
doh, yep.